src/cms/templates/pois/poi_form.html

```diff
@@ -109,14 +109,18 @@ <h1 class="heading">
                         <span class="inline-block mb-2 font-bold">{% trans 'Status' %}:</span>
                         {{ poi_translation_form.instance.get_status_display }}
                     {% endif %}
-                    <label for="{{ poi_translation_form.title.id_for_label }}" class="block mb-2 mt-4 font-bold">{{ poi_translation_form.title.label }}</label>
+                    <label for="{{ poi_translation_form.title.id_for_label }}" 
+                    data-slugify-url="{% url 'slugify_ajax' region_slug=region.slug language_slug=language.slug %}{% if poi_form.instance.id %}?page={{ poi_form.instance.id }}{% endif %}"
```

Comment:
The `slugify_ajax` view only works for pages at the moment. So it will not check whether a POI exists with that slug, but whether a Page exist with that slug: https://github.com/Integreat/integreat-cms/blob/ef44282cb1cd16ddaa5fab7ceeefca47629adca3/src/cms/views/pages/page_actions.py#L902
In order for this to work for POIs, it needs some generalization. If you think this is too complicated for now, I will open a new issue for it. I admit that this part is not really as simple as a good first issue should be.

Reply:
I think it is still difficult for me and therefore would like you to opne a new issue.

Reply:
Ok, no problem! Then you don't need to change the permalink / slug section anymore, I will open a new issue.